azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/common/audiodevicelist/AudioDeviceListViewModel.kt

```diff
@@ -12,24 +12,20 @@ import kotlinx.coroutines.flow.StateFlow
 internal class AudioDeviceListViewModel(
     private val dispatch: (Action) -> Unit,
 ) {
-    private lateinit var audioDeviceSelectionStatusStateFlow: MutableStateFlow<AudioDeviceSelectionStatus>
-    private val displayAudioDeviceSelectionMenuStateFlow = MutableStateFlow(false)
+
+    lateinit var audioDeviceSelectionStatusStateFlow: MutableStateFlow<AudioDeviceSelectionStatus>
+    val bluetoothScoAvailableStateFlow = MutableStateFlow(false)
+    val displayAudioDeviceSelectionMenuStateFlow = MutableStateFlow(false)
 
     fun init(audioDeviceSelectionStatus: AudioDeviceSelectionStatus) {
         audioDeviceSelectionStatusStateFlow = MutableStateFlow(audioDeviceSelectionStatus)
     }
 
-    fun update(audioDeviceSelectionStatus: AudioDeviceSelectionStatus) {
+    fun update(audioDeviceSelectionStatus: AudioDeviceSelectionStatus, isBluetoothSCOAvailable: Boolean) {
         audioDeviceSelectionStatusStateFlow.value = audioDeviceSelectionStatus
+        bluetoothScoAvailableStateFlow.value = isBluetoothSCOAvailable
     }
 
-    fun getAudioDeviceSelectionStatusStateFlow(): StateFlow<AudioDeviceSelectionStatus> {
-        return audioDeviceSelectionStatusStateFlow
```

Comment:
Adam, we added these methods to make sure we are not returning MutableStateFlow. We return StateFlow.

Reply:
Ok, I'll put it back.

Reply:
restored casting/private/public visibility